### DISCUSSION TITLE
Is any chance to integrate with 3rd party js library, such as ag-grid

### DISCUSSION THREAD

**Discussion by chorylee** [reflex-dev/reflex#699] [Ideas]
AG Grid Community is fridendly to use with high performance, also free. (https://www.ag-grid.com)
If can integrate it, we can display intensive datatable in a more professional way.

**Comment by FHU-yezi** [comment]
Yes, you can wrapping any React component, see this: https://pynecone.app/docs/advanced-guide/wrapping-react

**Reply by robingie** [reply]
It would be great to have an in-depth explanation for this. Especially for beginners.

**Comment by SeqCrafter** [comment]
I also want to have IGV.js (https://github.com/igvteam/igv.js) in my web app, Is there any method on that

**Comment by picklelo** [comment]
We have this now! https://reflex.dev/blog/2024-09-25-using-ag-grid-in-reflex/
